demisto_sdk/commands/generate_docs/generate_playbook_doc.py

```diff
@@ -55,7 +56,7 @@ def generate_playbook_doc(input, output: str = None, permissions: str = None, li
         if limitations:
             doc.extend(generate_numbered_section('Known Limitations', limitations))
 
-        doc.append('<!-- Playbook PNG image comes here -->')
+        doc.append(f'![{_name}](Fill in the link to your image here please)')
```

Comment:
I don't think the please is necessary here - but if you chose to keep it it should be: `Please Insert the link to your image here.`